src/blog/2024/01/soc2.md

```diff
@@ -0,0 +1,57 @@
+---
+title: FlowFuse is now SOC 2 Type 1 Compliant
+subtitle: FlowFuse's Path to SOC 2 Type 1 Compliance - A Testament to Our Commitment to Securing Customer and User Data.
+date: 2024-01-15
+authors: ["grey-dziuba"]
+image: /blog/2024/01/images/soc-2-compliant.png
+tags:
+    - posts
+    - flowfuse
+    - news
+---
+
+
+SOC 2, governed by the American Institute of Certified Public Accountants (AICPA), is a crucial framework for organizations handling customer data. It involves an independent audit assessing whether a company's controls are effectively designed and operational. Achieving SOC 2 compliance is not just about meeting standards; it's a declaration of our unwavering commitment to data security and customer trust.
+
+<!--more-->
+
+# Improving Your Security Posture
+For FlowFuse, pursuing SOC 2 compliance was a natural step in enhancing our security posture. In today's landscape, where cybersecurity threats and data breaches are rampant, prioritizing information security is not optional but a necessity. Our SOC 2 audit was more than a procedural hurdle; it was a comprehensive validation of our robust controls and processes by an independent third party.
+
```

Comment:
I'd structure this like:

1. When an organization is professionalizing Node-RED deployments their vendor needs to meet SOC2 requirements
1. We've always taken your data seriously, and we document our policies in our open handbook to be scrutinized by vendors
2. Providing an independent audit is better and faster for our customers.
3. This also provided FlowFuse with insights what policies are missing

Reply:
```suggestion
At FlowFuse, we understand that professionalizing Node-RED deployments for our clients means adhering to the highest standards, including SOC 2 requirements. This commitment is at the core of our security philosophy. In a world rife with cybersecurity threats and data breaches, taking information security seriously isn't just an option—it's a critical necessity. Our SOC 2 audit was far more than just a procedural step. It represented a comprehensive, independent third-party validation of our robust controls and processes. We believe in transparency and accountability, which is why we document our policies in our open handbook, inviting scrutiny from vendors and reinforcing trust with our customers. Providing this level of independent audit not only serves our customers better and more efficiently but also offers FlowFuse valuable insights into enhancing our security measures and identifying any gaps in our policies. This proactive approach ensures we continue keeping your data safe and secure at all times.
```